dev/docker/Dockerfile.habana

```diff
@@ -1,4 +1,4 @@
-FROM vault.habana.ai/gaudi-docker/1.14.0/ubuntu22.04/habanalabs/pytorch-installer-2.1.1:latest
+FROM vault.habana.ai/gaudi-docker/1.15.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.0:latest
```

Comment:
Should we also update to 1.15.1 in line 15 `pip install git+https://github.com/HabanaAI/DeepSpeed.git@1.14.0`?

Reply:
updated.